tutorials/distributed_data_classification/fineweb-edu-ensebmle-classification.ipynb

```diff
@@ -0,0 +1,1248 @@
+       "3                    1.234375   \n",
+       "4                    0.135742   \n",
+       "5                    1.089844   \n",
+       "6                    0.479004   \n",
+       "7                    0.632324   \n",
+       "8                    1.295898   \n",
+       "9                    0.735840   \n",
+       "\n",
+       "                                                text  \\\n",
+       "0  Quantum computing is set to revolutionize the ...   \n",
+       "1  Investing in index funds is a popular strategy...   \n",
+       "2  Recent advancements in gene therapy offer new ...   \n",
+       "3  Online learning platforms have transformed the...   \n",
+       "4  Traveling to Europe during the off-season can ...   \n",
+       "5  Training regimens for athletes have become mor...   \n",
+       "6  Streaming services are changing the way people...   \n",
+       "7  Vegan recipes have gained popularity as more p...   \n",
+       "8  Climate change research is critical for develo...   \n",
+       "9  Telemedicine has become increasingly popular d...   \n",
+       "\n",
+       "   fineweb-nemotron-edu-score-int  fineweb-mixtral-edu-score-int  \\\n",
+       "0                              19                             19   \n",
+       "1                               5                              5   \n",
+       "2                              18                             18   \n",
+       "3                              14                             14   \n",
+       "4                               1                              1   \n",
+       "5                              12                             12   \n",
+       "6                               3                              3   \n",
+       "7                               7                              7   \n",
+       "8                              16                             16   \n",
+       "9                               9                              9   \n",
+       "\n",
+       "   fasttext-quality-score-int  \n",
+       "0                          19  \n",
+       "1                           5  \n",
+       "2                          18  \n",
+       "3                          14  \n",
+       "4                           1  \n",
+       "5                          12  \n",
+       "6                           3  \n",
+       "7                           7  \n",
+       "8                          16  \n",
+       "9                           9  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "scored_data.df = scored_data.df.map_partitions(map_score_columns, score_col_names, threshold_dict)\n",
+    "scored_data.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cf526dd-363f-4199-bb9d-2ea9b8897fae",
+   "metadata": {},
+   "source": [
+    "# Step 4: Compute the Final Ensembled Score\n",
```

Comment:
So a higher score is better, right? 19 is the best possible score?

Reply:
Yeah, So its a percentile based score going from `0 to 19` for each 5 percentile points. 19 being the best.